src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -47,7 +48,7 @@ def load_arguments(self, _):
                    validator=validate_cluster_resource_group)
         c.argument('fips_validated_modules', arg_type=get_three_state_flag(),
                    options_list=['--fips-validated-modules', '--fips'],
```

Comment:
No need to add argument default values in the help. The default value will be retrieved from the function signature.